app/experimenter/templates/experiments/detail_base.html

```diff
@@ -193,18 +193,60 @@ <h5 class="col mt-3">History</h5>
       <div class="row">
         <div class="col">
           {{ user }}
-
           {% for user_change in user_changes %}
+            {% if "Edited Experiment" in user_change.pretty_status %}
+            <p id="editExperiment{{user_change.id}}" data-toggle="modal" data-target="#changelog-modal{{user_change.id}}">
```

Comment:
Ah I see, it's not clear that it's clickable, probably a clearly clickable anchor tag is better, and the little 'info' circle is more informative than the arrow I think:

            <p>
              <a href="" id="editExperiment{{user_change.id}}" data-toggle="modal" data-target="#changelog-modal{{user_change.id}}">
                <span class="fas fa-info-circle"></span>
                Edited Experiment
              </a>
            </p>


<img width="185" alt="Screenshot 2019-08-13 11 20 32" src="https://user-images.githubusercontent.com/119884/62953911-67885500-bdbc-11e9-9567-67a46181b2b1.png">